rasa/nlu/featurizers/sparse_featurizer/ngram_featurizer.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import warnings
+
+from typing import Any, Dict, Optional, Text
+
+from rasa.nlu.featurizers.featurzier import Featurizer
+
+logger = logging.getLogger(__name__)
+
+
+class NGramFeaturizer(Featurizer):
```

Comment:
should this rather inherit from `CountVectorsFeaturizer `? It doesn't look like this contains all the functionality the previous implementation had

Reply:
the whole thing is deprecated

Reply:
it is there, just to raise an exception that it is deprecated

Reply:
It is not raising an exception, it Is printing a warning. Deprecation means it works like before but the user needs to act as it won’t continue working in the future.

Reply:
we did a warning, because we didn't want to waste time on updating smth that actually nobody uses

Reply:
True, we are not deprecating it, we are removing it. Will exchange "deprecation" by "removal" and raise a proper error.